src/test/kotlin/com/example/flashcardbackend/integrationtest/DeckGroupIntegrationTest.kt

```diff
@@ -0,0 +1,338 @@
+package com.example.flashcardbackend.integrationtest
+
+import com.example.flashcardbackend.deckgroup.DeckGroupCreate
+import com.example.flashcardbackend.requestbuilder.DeckGroupHttpRequestBuilder
+import com.example.flashcardbackend.deckgroup.DeckGroupUpdate
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.hamcrest.Matchers
+import org.junit.jupiter.api.*
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.MediaType
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.jdbc.Sql
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.*
+
+@SpringBootTest
+@ActiveProfiles("integrationTest")
+@AutoConfigureMockMvc
+@Sql(
+    value = [
```

Comment:
If you want to initialize your database by using one SQL script, there is no need to have a separate cleanup script. You can combine both scripts if you want.